scripts/check_if_pr_is_low_risk_test.py

```diff
@@ -17,18 +17,14 @@
 from __future__ import annotations
 
 import builtins
+from io import StringIO  # pylint: disable=import-only-modules
```

Comment:
Just import `io` and then do `io.StringIO`

Reply:
Done